pkg/smart/check_required_tools_linux.go

```diff
@@ -7,16 +7,28 @@ import (
 	"bytes"
 	"fmt"
 	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
 )
 
-func checkTools() (string, error) {
-	cmd := exec.Command("/bin/sh", "-c", "smartctl -h")
+const defaultSmartctlPath = "smartctl"
+
+func checkTools(smartctl string) (string, string, error) {
```

Comment:
Should be refactored and reused in _darwin.go

Reply:
I think linux and darwin should be separate

Reply:
Ok.

Reply:
I'm not saying you should drop this file. I mean it would be preferred to refactor the common code instead of repeating it.

Reply:
Let's consider this in a upcoming PR instead